src/test/java/edu/cornell/kfs/module/purap/batch/service/impl/JaggaerGenerateContractPartyCsvServiceImplTest.java

```diff
@@ -97,9 +107,10 @@ private JaggaerContractAddressUploadDto buildNoVendorAddressDto() {
     }
 
     @Test
-    void testGetJaggerContractsDto() {
+    void testGetJaggerContractsDto() throws ParseException {
```

Comment:
Another method with a minor typo that should be fixed: "Jagger" --> "Jaggaer"

Reply:
ok